modules/ROOT/pages/otel-support.adoc

```diff
@@ -0,0 +1,145 @@
+
+=== Example of Flow Tracing Feature
+
+When tracing a flow execution, each Mule component execution involved in processing the message is represented as spans. The component spans describe the relevant aspects of the execution, which can range from a single span for simple components, such as the Set Payload, to more complex structures, such as a Batch Job.
+
+.Batch Instance 
+The Mule runtime traces complex components, such as Batch Jobs.
+
+image::otel-batch-instance.png["A batch flow including a Listener and Set Payload components, and a more complex structure"]
+
+.Spans Generated
+Each batch record processed by the first step of the Batch Job is shown in the trace with each record span containing a child span representing the unit of work done by the Logger component.
+Then, the Batch Aggregator span is shown, along with the For Each and the processors executed within it, such as the Transform Message and the Logger. 
+Following this, the trace displays the other two batch steps, along with their respective components.
+Finally, the Batch On Complete span is shown, along with the Logger component that appears after the Batch Job in the flow. 
+
+=== Example of Distributed Tracing Feature
+
+Consider two Mule apps where the first sends a request to the second.
+
+image::otel-mule-app1.png["A flow including a Listener, Logger, and Request components of an HTTP app"]
+
+image::otel-mule-app2.png["A flow showing the process flow for an asynchronous application"]
+
+Even though these are two different flows, distributed tracing captures both as part of the same trace because OpenTelemetry is instrumentation in Mule runtime and HTTP Connector.
+
+=== Errors in Tracing
+
+If the execution of the unit of work represented by a span produces an error, the span's status is set to `Error`. MuleSoft follows the semantic convention defined by OpenTelemetry that adds one or more `exception` events to the span:
+
+* `exception.type`: Mule error type
+* `exception.message`: Mule error message
+* `exception.stacktrace`: Mule flow stack string representation
+
+In the following example, when an error is produced, the span corresponding to the Raise Error component shows an error status in the Lightstep platform. 
+
+image::otel-errors-in-tracing.png["A flow of a telemetry flow with error handling"]
+
+This status is propagated to the parent span, for example, the flow span. A detailed view of the span shows the error status and the exception events.
+
+=== HTTP Conventions
+
+The HTTP Listener and the HTTP Request operations contribute additional attributes to the spans, providing more context for tracing the HTTP interactions.
+
+For flows with an HTTP Listener:
+
+* Span name: Combination of HTTP Method and path. For example, `POST /api`.
+* Span kind: `SERVER`.
+
+Additional attributes for an HTTP Listener associated flow span:
+
+[%header%autowidth.spread]
+|===
+| Attribute | Description 
+| `net.host.name` | The host address of the HTTP Listener
```

Comment:
You can remove the article from the start of all these descriptions (streamline for easier scanning).